### PR TITLE
Create dms-der-moderne-staat.csl

### PR DESCRIPTION
**Pull request by shaunss**
Citation style for the German political science journal "dms - der moderne staat" (http://www.budrich-journals.de/index.php/dms)

### CODE REVIEW

**Comment by rmzelle** [comment]
Can you follow the instructions at https://help.github.com/articles/moving-a-file-to-a-new-location/ and move your style out of the "dependent" subdirectory? Start by clicking the pencil icon at https://github.com/shaunss/styles/blob/patch-2/dependent/dms-der-moderne-staat.csl

**Comment by adam3smith** [comment]
http://www.budrich-journals.de/index.php/dms/about/submissions#authorGuidelines
should replace the link to the APSA guidelines as the documentation link.

**Comment by rmzelle** [comment]
The `<rights/>` line should be `<rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>` as well.

File: der-moderne-staat.csl [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" collapse="year">
+    <sort>
+      <key macro="year-date"/>
+      <key macro="author-short"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=", ">
+        <group delimiter=" ">
+          <text macro="author-short"/>
```
Comment (zuphilip) [inline der-moderne-staat.csl]: Should be <i>italic</i>, cf. "Namen von Personen im laufenden Text und im Literaturverzeichnis bitte kursiv."

File: der-moderne-staat.csl [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+      <date variable="issued" prefix=", " suffix=":">
+        <date-part name="year"/>
+      </date>
+      <choose>
+        <if type="bill legal_case report" match="any">
+          <text macro="legal_case"/>
+          <group prefix=" " delimiter=" ">
+            <text macro="title" suffix=","/>
+            <text variable="number" suffix=","/>
+          </group>
```
Comment (zuphilip) [inline der-moderne-staat.csl]: That looks a little unstable in cases where one of the two variables/macro is empty. Try to move the affixed to the group instead, i.e.:
```
<group prefix=" " delimiter=", " suffix=",">
  <text macro="title"/>
  <text variable="number" />
</group>
```

**Comment by zuphilip** [comment]
Are you sure the date format is correct? I have never seen something like "Januar 12, 2005" in German text.

**Comment by zuphilip** [comment]
It looks like you are missing the "in: " for journal articles, e.g. from the guidelines:

<i>Müller, Rudi/Schmidt, Cäsar</i>, 1981: Bedingungen jugendlicher Delinquenz, in: Zeitschrift für
Kindheitsentwicklung, 20, S. 431-443.

File: der-moderne-staat.csl [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    <category citation-format="author-date"/>
+    <category field="political_science"/>
+    <issnl>1865-7192</issnl>
+    <summary>der moderne staat</summary>
+    <updated>2015-01-29T12:46:57+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="editor" form="short">Hrsg.</term>
```
Comment (zuphilip) [inline der-moderne-staat.csl]: That should already be the standard in the local file and you don't need to define it here seperately.

File: der-moderne-staat.csl [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+            <text variable="container-title" font-style="normal"/>
+            <group delimiter=", ">
+              <text variable="volume"/>
+              <text variable="issue" form="short"/>
+            </group>
+            <text variable="page" form="short" prefix="S. "/>
+          </group>
+        </else>
+      </choose>
+      <text prefix=" " macro="access" suffix="."/>
```
Comment (zuphilip) [inline der-moderne-staat.csl]: The `suffix="."` seems to be too much, because the `layout` has already a suffix.

**Comment by adam3smith** [comment]
See the additional review requests by @zuphilip above. Update instructions are the same. 
Let us know if you have any questions. 
Also let us know when you're done, we don't get notified on updated pull requests.

**Comment by shaunss** [comment]
I can't find this in the csl file. Where did you see this?

on 31.01.15 14:22 Philipp Zumstein wrote:
>
> Are you sure the date format is correct? I have never seen something like "Januar 12, 2005" in German text.
>

**Comment by adam3smith** [comment]
lines 203 and 216 in the current version.

**Comment by zuphilip** [comment]
This seems fine now AFAIS.

**Comment by rmzelle** [comment]
Oh, @shaunss, can you replace

```
<issnl>1865-7192</issnl>
```

with

```
<issn>1865-7192</issn>
<eissn>2196-1395</eissn>
```

please? The ISSN-L hasn't really caught on, so we prefer having the print and electronic ISSN.

**Comment by rmzelle** [comment]
Great, thanks!

**Pull request merged**